tests/unit/StandaloneDirectorsFiling.spec.ts

```diff
@@ -388,9 +388,28 @@ describe('Standalone Directors Filing - Part 2A - Resuming with FAS staff paymen
     // verify that we stored the Filing ID
     expect(+vm.filingId).toBe(123)
 
-    // verify that we loaded the director data correctly
-    expect(vm.filingData.filter(el => el.filingTypeCode === 'OTCDR').length).toEqual(1)
```

Comment:
As the component under test was shallow-mounted, the Directors sub-component does not emit events to update the filing data.
So I moved these checks out of here and into their own test below (where I explicitly call the event handlers).